src/constants/blockchain.ts

```diff
@@ -72,7 +119,6 @@ export const CHAINS: Record<ChainId, ChainMetadata> = {
     name: "Polygon",
     chainId: ChainId.POLYGON,
     logoURI: polygonLogo,
-    rpcUrl: "https://polygon-rpc.com/",
```

Comment:
This was removed since RPC URLs go in an env variable now?

Reply:
this was an unecessary value when copying the block from another code base.  the rpc url gets added through envs in the `oracleConfig` file